week1/project/ecommerce/src/components/Categories.jsx

```diff
@@ -0,0 +1,18 @@
+import allCategories from "../fake-data/all-categories"
+
+import Category from "./Category"
+
+function Categories() {
+  
+  return (
+    <div className="categories_container"> 
+      {
+        allCategories.map((category, index) => (
```

Comment:
Same for the use of index :)

Reply:
categories array, has no ids and we are allowed to edit it,
once we work with the API I will change it to the category id